src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -171,6 +166,7 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
         StrategyAndWeightingMultiplier[] memory _strategiesConsideredAndMultipliers
     ) internal {
         uint8 quorumNumber = quorumCount;
+        require(quorumNumber < 255, "VoteWeigherBase._createQuorum: number of quorums cannot 256");
```

Comment:
typo: "cannot 256" => "cannot exceed 256" ?

Also shouldn't this be 256 instead of 255? This was the max you'll hit is 255 -- maybe you were thinking of using non-strict inequality in the check when using 255?

Also probably want to define a constant like `MAX_QUORUMS = 256` and then use it instead of a "magic number"

Reply:
great catch: https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/d4be2b7f8585ad8ef9171fba9171db253b572e0b